pkg/kv/kvserver/replica_follower_read.go

```diff
@@ -71,9 +71,18 @@ func BatchCanBeEvaluatedOnFollower(ctx context.Context, ba *kvpb.BatchRequest) b
 				return false
 			}
 		case r.Method() == kvpb.Export:
-		// Export requests also have clear semantics when served under the closed
-		// timestamp as well, even though they are non-transactional, as they
-		// define the start and end timestamp to export data over.
+			// Export requests also have clear semantics when served under the closed
+			// timestamp as well, even though they are non-transactional, as they
+			// define the start and end timestamp to export data over.
+			if r.(*kvpb.ExportRequest).ExportFingerprint {
+				// Fingerprinting directly sends Export Requests to the gateway node as opposed
```

Comment:
/to the/from the/

Reply:
From the gateway node? I thought the idea was the gateway does all the work.

Reply:
that's the outcome, yes. but the reason that happens is that it is because the gateway _sends_ all the requests (and then sends them to itself, since it is closest). The sentence here is talking about why fingerprints are different that backups, and difference is where we send the requests from (with the result being where they get sent to).

It's a subtle distinction I suppose